Lesson_9_Modeling/Lesson_9_Modeling.ipynb

```diff
@@ -0,0 +1,4606 @@
+{
```

Comment:
<p>-change lets to let's throughout document</p><p>-not super critical but being consistent with headers would be nice (capitalized vs not) but not too important</p><p> </p><p>-For some reason my jupyter notebook would not execute... didn't keep trying because my computer died :'(</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>